tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -147,6 +157,8 @@ public void start() throws MQClientException {
 
                 mqClientInstance.start();
 
+                executors = Executors.newCachedThreadPool(new ThreadFactoryImpl("MQAdminThread_"));
```

Comment:
IMO，it's better to specify pool core size, max size, length of waiting list and reject conduct.